ci-operator/jobs/openshift/release/openshift-release-master-presubmits.yaml

```diff
@@ -954,32 +954,6 @@ presubmits:
           requests:
             cpu: 10m
     trigger: (?m)^/test( | .* )jira-lifecycle-config,?($|\s.*)
-  - agent: kubernetes
```

Comment:
This needs to be removed in another PR in advanced.

Reply:
Ok so we should open another PR, remove this presubmit and then come back and keep working on this.

Reply:
we should merge this PR right after the presubmit removal.